docs/data/date-pickers/custom-components/ArrowSwitcherComponent.tsx

```diff
@@ -15,7 +15,7 @@ import { StaticTimePicker } from '@mui/x-date-pickers/StaticTimePicker';
 import { StaticDateRangePicker } from '@mui/x-date-pickers-pro/StaticDateRangePicker';
 import { DateRange } from '@mui/x-date-pickers-pro';
 
-const components: DatePickerProps<any>['components'] = {
+const components: Omit<DatePickerProps<any>['components'], 'Day'> = {
```

Comment:
The signature of the day slot is not the same for range and non-range pickers so we had a TS error here
But since we don't use this slot, we can just omit it